stdlib/public/Python/NumpyConversion.swift

```diff
@@ -0,0 +1,180 @@
+//===-- NumpyConversion.swift ---------------------------------*- swift -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file defines the `NumpyArrayConvertible` protocol for bridging
+// `numpy.ndarray`.
+//
+//===----------------------------------------------------------------------===//
+
+/// A type that can be initialized from a `numpy.ndarray` instance represented
+// as a `PyValue`.
+public protocol NumpyArrayConvertible {
```

Comment:
This name is misleading because `XXConvertible` types generally imply that the type can be **converted to XX**, not initialized from it. For example, `CustomStringConvertible` types can be converted to `String`, not vice-versa.

Other names that are closer to the intended meaning include:
* ExpressibleByNumpyArray
* NumpyArrayInitializable

Reply:
I think `ConvertibleFromNumpyArray` is the best name! I'll use it.